docs/time/period.rst

```diff
@@ -0,0 +1,23 @@
+.. _time:
+*******************************************
+Period detection  (`gammapy.time.lombscargle`) 
+*******************************************
```

Comment:
There's a few formatting issues with your `docs/time/period.rst` file.

In order to be able to resolve those, you need to run the Sphinx docs build locally and fix the errors / warnings, and look at the generated HTML docs, using these commands:
```
python setup.py build_docs
open docs/_build/html/index.html
```

Since you're introducing a new sub-page besides `index.rst` in the `gammapy.time` docs, you have to add something to `docs/time/index.rst` so that the new page gets linked to and becomes part of the docs.

An example is here:
https://github.com/gammapy/gammapy/blame/master/docs/spectrum/index.rst#L75
so adding this at the end of `docs/time/index.rst` (before the API ref) should do the trick:
```
Content
=======
.. toctree::
   :maxdepth: 1

   period
```